wordcount1.c

```diff
@@ -0,0 +1,24 @@
+//Matthew Kellerman -prints how many words there are in text file
+#include <stdio.h>
```

Comment:
Nice job, but you didn't take newlines or whitespaces into consideration, so the count was off!